Libraries/Image/NativeImageEditor.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+type cropOptions = {
+  offset: {|
+    x: number,
+    y: number,
+  |},
+
+  size: {|
+    width: number,
+    height: number,
+  |},
+
+  displaySize?: ?{|
+    width: number,
+    height: number,
+  |},
+
+  resizeMode?: ?$Enum<{
```

Comment:
are these optional because android doesn't have them? if so, group and mark: `// iOS only`